sros2/sros2/api/__init__.py

```diff
@@ -236,24 +270,18 @@ def create_keystore(keystore_path):
         print('directory already exists: %s' % keystore_path)
 
     ca_conf_path = os.path.join(keystore_path, 'ca_conf.cnf')
+    ca_key_path = os.path.join(keystore_path, 'ca.key.pem')
+    ca_cert_path = os.path.join(keystore_path, 'ca.cert.pem')
+
     if not os.path.isfile(ca_conf_path):
         print('creating CA file: %s' % ca_conf_path)
         create_ca_conf_file(ca_conf_path)
     else:
         print('found CA conf file, not writing a new one!')
 
-    ecdsa_param_path = os.path.join(keystore_path, 'ecdsaparam')
-    if not os.path.isfile(ecdsa_param_path):
-        print('creating ECDSA param file: %s' % ecdsa_param_path)
-        create_ecdsa_param_file(ecdsa_param_path)
```

Comment:
Do we no longer need this, then? What purpose was it serving?

Reply:
This file was the "elliptic curve parameters" for creating the cert. Since we're building in Python, we don't need to write out a parameter file to use as an input to generate the private key for the CA cert, we just specify in the call here https://github.com/ros2/sros2/pull/126/files#diff-4db4152194feafaa6ef7a7fd1c647c53R218 - instead of writing to a file here https://github.com/ros2/sros2/pull/126/files#diff-4db4152194feafaa6ef7a7fd1c647c53R214

Reply:
Excellent, thank you.